charts/aws-ebs-csi-driver/templates/NOTES.txt

```diff
@@ -1,3 +1,43 @@
 To verify that aws-ebs-csi-driver has started, run:
 
     kubectl get pod -n {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "aws-ebs-csi-driver.name" . }},app.kubernetes.io/instance={{ .Release.Name }}"
+
+
+WARNING: The following values have been deprecated in favor of moving them into the controller or node groups.  They will be removed in a subsequent release.
+
+affinity:
+enableVolumeScheduling:
+enableVolumeResizing:
+extraCreateMetadata:
+extraVolumeTags:
+k8sTagClusterId:
+nodeSelector:
+podAnnotations:
+priorityClassName:
+region:
+replicaCount:
+resources:
+tolerations:
+topologySpreadConstraints:
+volumeAttachLimit:
+
+are moving to
+
+controller:
+  affinity:
+  enableVolumeScheduling:
+  enableVolumeResizing:
+  extraCreateMetadata:
+  extraVolumeTags:
+  k8sTagClusterId:
+  nodeSelector:
+  podAnnotations:
+  priorityClassName:
+  region:
+  replicaCount:
+  resources:
```

Comment:
what about resources vs containerResources, can we deprecate resources in favour of containerResources?

Reply:
We could.  The way it is written currently is that specifying controller.resources will apply to all containers in the deployment, but they can be overridden by setting the individual container resources.  If we don't like that we can decide to get rid of controller.resources on my next PR that makes the breaking changes.

Reply:
okay, it sounds useful then, like i im agine people might want to set some arbitrary nonzero resource.requests for all containers as a starting point and tune for individual containers from there